### ISSUE TITLE
The "autoFocus" and "trapFocus" does not work when the focus element, in the modal window, is hidden.

### ISSUE THREAD

**Issue by pleomax55** [fancyapps/ui#139]
When an empty modal window opens and there is an element in it that can be in focus BUT it is hidden, then the "autoFocus" and "trapFocus" simply does not work, and the focus remains outside the modal window.

I use this code:
```
(function ($) {
  var Fancybox_opt = {
    dragToClose: false,
    mainClass: "simple-modal",
    showClass: "fancybox-fadeIn",
    hideClass: "fancybox-fadeOut"
  };
  Fancybox.bind("[data-src='#simple-modal']", Fancybox_opt);
  // or
  $("body").on("click", ".open-sm", function () {
    Fancybox.show([{ src: "#simple-modal", type: "inline" }], Fancybox_opt);
  });
})(jQuery);
```
HTML:
```
<a href="javascript:;" data-src="#simple-modal">open #1</a>
<a href="javascript:;" class="open-sm">open #2</a>
<button>simple button</button>

<div id="simple-modal" style="display:none;">
  <div class="d-none">
    <button>hidden button</button>
  </div>
</div>
```
CSS:
```
.d-none { display: none; }
:focus { outline: 2px solid red; outline-offset: 3px; } /* <-- for better visibility of the focus in the example  */
```

Codepen example: //codepen.io/pleomax55/pen/BaZeLOQ

As a result, keyboard navigation does not work correctly.

**Comment by pleomax55** [comment]
sorry @fancyapps, but I don't think the bug has been fixed.
take a look here https://codepen.io/pleomax55/pen/BaZeLOQ
I used the latest version and the focus remained outside the modal window.
I added an additional button inside the window to check if the focus will switch to the next element, but he didn't do it.
a hidden button, or any hidden element that may be in focus, breaks "autofocus" and "trap Focus"
